spec/widgets/histogram/content-view.spec.js

```diff
@@ -12,7 +12,8 @@ describe('widgets/histogram/content-view', function () {
     });
 
     this.widgetModel = new HistogramWidgetModel({
-      title: 'Howdy'
+      title: 'Howdy',
+      attrsNames: ['title']
```

Comment:
I must put otherwise the `this.widgetModel.update()` (see https://github.com/CartoDB/deep-insights.js/blob/221-update-hist-title/spec/widgets/histogram/content-view.spec.js#L112) do not update the title.